components/script/dom/domimplementation.rs

```diff
@@ -80,7 +80,7 @@ impl DOMImplementationMethods for DOMImplementation {
         let doc = XMLDocument::new(win,
                                    HasBrowsingContext::No,
                                    None,
-                                   self.document.origin().alias(),
```

Comment:
Since the spec does not talk about aliasing any more there is no need for a special name here.